README.md

```diff
@@ -105,7 +104,6 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - [What open-source should I contribute to](https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to/) by Kent C. Dodds
 
 ## Using Version Control
-- [Think Like (a) Git](http://think-like-a-git.net/) - Git introduction for "advanced beginners," but are still struggling, in order to give you a simple strategy to safely experiment with git.
```

Comment:
Similar to your other PR https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/455, this link still works for me.

Reply:
Yeah we can add it.